library/lib_number_string.ks

```diff
@@ -0,0 +1,46 @@
+// This file is distributed under the terms of the MIT license, (c) the KSLib team
+
+@LAZYGLOBAL off.
+
+function numString {
+ parameter
+  number,  //input number
+  ip,      //number of digits before the decimal point.
```

Comment:
For some reason Pascal, Python and C++ use different approach: total number of characters and number of decimal places.